app/models/micropost.rb

```diff
@@ -0,0 +1,15 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  scope :order_desc, ->{order created_at: :desc}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
```
Tìm hiểu xem 2 lệnh này khác gì nhau và nên dùng cái nào nhé

Reply:
validates :user yêu cầu user đó tồn tại trong bảng users, còn validates :user_id thì không, ở đây nên dùng validates :user